client_src/js/lib/octoprint.js

```diff
@@ -308,17 +310,42 @@ export default class OctoPrintClient {
                 "printer/tool",
                 offset
               );
+              console.log(offset);
             }
             await OctoPrintClient.post(printer, "printer/printhead", feed);
             const post = await OctoPrintClient.post(printer, "job", opts);
-            element.target.disabled = false;
+            if (element) {
+              element.target.disabled = false;
+            }
           }
         },
       });
     } else {
       await OctoPrintClient.post(printer, "printer/printhead", feed);
       const post = await OctoPrintClient.post(printer, "job", opts);
-      element.target.disabled = false;
+      if (
+        printer.selectedFilament != null &&
```

Comment:
Type coercion with undefined/null/falsy due to == instead of ===

Could benefit of more explicit checking.